### PR TITLE
Bundled default and tuned samples into ml product

### PR DESCRIPTION
**Pull request by manoramahp**

### CODE REVIEW

File: modules/samples/default/decision-tree/README.MD [hunk 1]
```diff
@@ -7,7 +7,9 @@ Algorithm: Decision-Tree
 Before Running
 ==============
 
-* Start WSO2 Machine Learner product with a fresh database.
+* Start WSO2 Machine Learner product.
+
+* Make sure jq is installed.
```
Comment (nirmal070125) [inline modules/samples/default/decision-tree/README.MD]: @manoramahp what is jq?

File: modules/samples/default/decision-tree/README.MD [hunk 2]
```diff
@@ -19,4 +21,4 @@ Outcome
 
 * Running the above script will result in a generation of logistic regression model for the provided data-set.
 
-* Model will be stored in the file system at /tmp directory.
+* Model will be stored in the file system at {ML_HOME}/tmp directory.
```
Comment (nirmal070125) [inline modules/samples/default/decision-tree/README.MD]: Isn't it under {ML_HOME}/models ?

File: modules/samples/default/decision-tree/README.MD [hunk 3]
```diff
@@ -19,4 +21,4 @@ Outcome
 
 * Running the above script will result in a generation of logistic regression model for the provided data-set.
```
Comment (nirmal070125) [inline modules/samples/default/decision-tree/README.MD]: Can we fix the algorithm names in READMEs @manoramahp ?

File: modules/samples/default/decision-tree/README.MD (hunk 1 above)
Comment (manoramahp) [inline modules/samples/default/decision-tree/README.MD]: jq command line tool has been used in the model-generation.sh script to read values from the json responses. (ie. to get the projectId, analysisId, etc.)

File: modules/samples/default/decision-tree/README.MD (hunk 2 above)
Comment (manoramahp) [inline modules/samples/default/decision-tree/README.MD]: Yes, it should be {ML_HOME}/models. I'll correct it.

File: modules/samples/default/decision-tree/README.MD (hunk 3 above)
Comment (manoramahp) [inline modules/samples/default/decision-tree/README.MD]: Fixed in https://github.com/wso2/product-ml/commit/f25ef0e5d5191348e2ef784499d156480a1d3dc0

File: modules/samples/default/decision-tree/README.MD (hunk 1 above)
Comment (nirmal070125) [inline modules/samples/default/decision-tree/README.MD]: @manoramahp thanks. We don't have any other alternative right? If it is the case, can we at least give a link to how to install it etc. for different OSs.

Comment (manoramahp) [inline modules/samples/default/decision-tree/README.MD]: Added the link in readme files : https://github.com/wso2/product-ml/commit/81c096aa2e69d968d530df9a2cc08e8cbca72847

**Pull request merged**